opal/mca/threads/pthreads/threads_pthreads_mutex.h

```diff
@@ -115,8 +115,6 @@ OPAL_DECLSPEC OBJ_CLASS_DECLARATION(opal_recursive_mutex_t);
     }
 #endif
 
-#endif
```

Comment:
Here I should not remove this `#endif`. I have to check qthreads and argobots too.